client/state/action-types.js

```diff
@@ -340,6 +340,7 @@ export const JETPACK_MODULE_ACTIVATE_SUCCESS = 'JETPACK_MODULE_ACTIVATE_SUCCESS'
 export const JETPACK_MODULE_DEACTIVATE = 'JETPACK_MODULE_DEACTIVATE';
 export const JETPACK_MODULE_DEACTIVATE_FAILURE = 'JETPACK_MODULE_DEACTIVATE_FAILURE';
 export const JETPACK_MODULE_DEACTIVATE_SUCCESS = 'JETPACK_MODULE_DEACTIVATE_SUCCESS';
+export const JETPACK_ONBOARDING_CREDENTIALS_RECEIVE = 'JETPACK_ONBOARDING_CREDENTIALS_RECEIVE';
```

Comment:
Nitpick -- I'd call it `JETPACK_ONBOARDING_RECEIVE_CREDENTIALS`, which is a bit easier on the eye.

Rationale: I'd consider `JETPACK_ONBOARDING` the domain specific prefix, and `RECEIVE_CREDENTIALS` the 'verb'.

Since we're using the data layer, we're also unlikely to need `_FAILURE` and `_SUCCESS` actions, are we?

Reply:
> Since we're using the data layer, we're also unlikely to need _FAILURE and _SUCCESS actions, are we?

That's right 👍 

> I'd call it JETPACK_ONBOARDING_RECEIVE_CREDENTIALS, which is a bit easier on the eye.

Oh well, I kind of disagree with that - `RECEIVE` is the verb, and `CREDENTIALS` is the context, so I'm more inclined to keep it as-is, in order to be consistent with the rest of the action names, where we keep the actual verb at the end.

Reply:
:man_shrugging: Your call 😄 

BTW I just realized we're not actually using the data layer here (but the point about not needing `_FAILURE` and `_SUCCESS` still holds).

Reply:
Yeah, we'll be using the data layer for different stuff (saving and retrieving step data)